Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -723,20 +723,28 @@ static void registerLogHook()
         if (msg->buffer_sz > 1024)
             return;
 
-        // Skip faults and debug logs in non-internal builds.
-        if (type == OS_LOG_TYPE_FAULT || type == OS_LOG_TYPE_DEBUG)
+        // Skip debug logs in non-internal builds.
+        if (type == OS_LOG_TYPE_DEBUG)
             return;
 
         CString logFormat(msg->format);
         CString logChannel(msg->subsystem);
         CString logCategory(msg->category);
 
+        auto qos = Thread::Background;
+
+        // Send fault logs with high priority. If the WebContent process is terminated, we might not be able to send the log in time.
+        if (type == OS_LOG_TYPE_FAULT) {
+            type = type == OS_LOG_TYPE_ERROR;
```

Comment:
What's this?

Reply:
That was a typo, fixed in latest patch :)